modules/core/fieldkit/farm_fieldkit.post_update.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for farm_fieldkit module.
+ */
+
+use Drupal\simple_oauth\Oauth2ScopeInterface;
+
+/**
+ * Enable simple oauth password grant.
+ */
+function farm_fieldkit_post_update_enable_password_grant(&$sandbox = NULL) {
```

Comment:
Do we need to make sure this runs after `farm_api_post_update_enable_static_oauth2_scopes()`? If it runs first, then the `static` scope provider won't be enabled, and the default `['farm_manager', 'farm_worker']` scopes won't load, and then won't be added to the `fieldkit` consumer. I think anyway...

Reply:
Hmm yes I think you are right. And taking a step back, we should make sure the `simple_oauth` hooks run before `farm_api` as well. Does this mean we can't use `hook_post_update_NAME` ?

Reply:
Aha, but if we use `post_update`, we can ensure they run after `simple_oauth` normal update hooks :+1:

Reply:
We also identified that `farm_fieldkit_post_update_*` should run after `farm_api_post_update_*` simply due to alphabetical. I will test this soon and confirm.